### PR TITLE
fix: Access token should be hidden/masked by default in Agent Dashboard

### PR DESCRIPTION
**Pull request by gitstart**
Fixes https://github.com/chatwoot/chatwoot/issues/6322

## Description

Supports masking/unmasking sensitive data such as API Tokens in the agent dashboard.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Click on user avatar, and select `profile settings` option.
- Scroll to bottom of page, and verify that the API token is hidden

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

https://user-images.githubusercontent.com/45232708/219290721-c67f1b15-8509-4c33-a8cc-2f2bdae97389.mov

### CODE REVIEW

**Comment by gitstart** [comment]
Hi @scmmishra @sojan-official 
The `Code` component design specs was used here, to create a maskable text field, while maintaining the  usage of `highlight.js` internally.

To enhance its reusability for other potential use cases, What are your thoughts on these suggestions:

- Eliminate the use of highlight.js within this component, as it would not ideally be used for rendering code snippets.
- Incorporate icons for eyes and copy at the end of a textbox to render maskable fields.

File: app/javascript/dashboard/components/MaskedText.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="text--container">
+    <button class="button small button--text" @click="onCopy">
+      {{ $t('COMPONENTS.CODE.BUTTON_TEXT') }}
+    </button>
+    <woot-button
+      variant="clear"
+      class="small button--visibility"
```
Comment (nithindavid) [inline app/javascript/dashboard/components/MaskedText.vue:8]:
```suggestion
    <woot-button size="small" class=" button--text" @click="onCopy">
      {{ $t('COMPONENTS.CODE.BUTTON_TEXT') }}
    </woot-button>
    <woot-button
      variant="clear"
      size="small"
      class="button--visibility"
      color-scheme="secondary"
```

File: app/javascript/dashboard/components/MaskedText.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  .button--text {
+    margin-top: 0;
+    position: absolute;
+    right: 0;
+  }
+
+  .button--visibility {
+    margin-top: 0;
+    position: absolute;
```
Comment (nithindavid) [inline app/javascript/dashboard/components/MaskedText.vue:60]:
```suggestion
```

File: app/javascript/dashboard/components/MaskedText.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.text--container {
+  position: relative;
+  text-align: left;
+
+  .button--text {
```
Comment (nithindavid) [inline app/javascript/dashboard/components/MaskedText.vue:52]:
```suggestion
  .button--text,
  .button--visibility {
```

**Review by nithindavid — APPROVED** [review]
@raph941 Great PR 🔥

**Pull request merged**